### PR TITLE
Add release note for Microsoft.FeatureManagement 3.2.0

### PR DESCRIPTION
**Pull request by zhiyuanliang-ms**

### CODE REVIEW

**Review by jimmyca15 — APPROVED** [review]

File: releaseNotes/Microsoft.Featuremanagement.md [hunk 1]
```diff
@@ -4,6 +4,16 @@
...
 
+## 3.2.0 - February 29, 2024
+
+### Enhancements
+
+* Supported declaring feature flags with [Microsoft Feature Management schema](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureManagement.v1.0.0.schema.json)
+
+### Breaking Changes
+
+There are no breaking changes in this release.
```
Comment (zhenlan) [inline releaseNotes/Microsoft.Featuremanagement.md:15]: I can understand we made this statement in the previous release (4.0.0-preview), which is a major version update, but are we going to say this in every new release going forward? Feels like over kill.

File: releaseNotes/Microsoft.Featuremanagement.md [hunk 2]
```diff
@@ -4,6 +4,16 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 3.2.0 - February 29, 2024
+
+### Enhancements
+
+* Supported declaring feature flags with [Microsoft Feature Management schema](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureManagement.v1.0.0.schema.json)
```
Comment (zhenlan) [inline releaseNotes/Microsoft.Featuremanagement.md:11]: Sorry for the nit, but the FM library doesn't declare feature flags. It reads feature flags declared by someone else. Does this work?

Added support for feature flags defined using [Microsoft Feature Management schema](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureManagement.v1.0.0.schema.json).

Comment (zhiyuanliang-ms) [reply releaseNotes/Microsoft.Featuremanagement.md:11]: Sounds good.

File: releaseNotes/Microsoft.Featuremanagement.md (hunk 1 above)
Comment (zhiyuanliang-ms) [reply releaseNotes/Microsoft.Featuremanagement.md:15]: Removed.

**Review by zhenlan — APPROVED** [review]

**Pull request merged**